lib/replace.js

```diff
@@ -116,8 +116,9 @@ module.exports = function (files, pattern, opts) {
     glob(locations, {mark: true, dot: true}, function (error, location) {
       if (error) return cb(error);
       location = _.uniq(location);
-      location = _.filter(location, function (item) {
-        return item.lastIndexOf('/') !== item.length - 1;
+      // Filter out all directories and .git .hg
+      location = _.filter(location, function (file) {
+        return file.lastIndexOf('/') !== file.length - 1 && file.indexOf('.git/') !== 0 && file.indexOf('.hg/') !== 0;
```

Comment:
I suggest using `_.startsWith`

Reply:
not supported in used lodash version

Reply:
Can we update the lodash version?

Reply:
We could, but it will involve a lot of testing and actions for rebuild and changes

Reply:
Ok, in that case please create two functions: startsWith and endsWith to improve readability